quazip/JlCompress.h

```diff
@@ -42,6 +42,22 @@ see quazip/(un)zip.h files for details. Basically it's the zlib license.
   */
 class QUAZIP_EXPORT JlCompress {
```

Comment:
Do these names and values come from minizip? If so, add a comment for posterity.

Reply:
The values in the CompressionStorage enumeration all refer to those in the zip.h file.